lib/node_modules/@stdlib/constants/float32/half-pi/lib/index.js

```diff
@@ -0,0 +1,52 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+/**
+* One half times the mathematical constant `π`.
+*
+* @module @stdlib/constants/float32/half-pi
+* @type {number}
+*
+* @example
+* var FLOAT32_HALF_PI = require( '@stdlib/constants/float32/half-pi' );
+* // returns 1.5707963705062866
+*/
+
+// MODULES //
+
+var float64ToFloat32 = require( '@stdlib/number/float64/base/to-float32' );
+
+
+// MAIN //
+
+/**
+* One half times the mathematical constant `π`.
+*
+* @constant
+* @type {number}
+* @default 1.5707963705062866
+* @see [Wikipedia]{@link https://en.wikipedia.org/wiki/Pi}
+*/
+var FLOAT32_HALF_PI = float64ToFloat32( 1.5707963705062866 );
```

Comment:
```suggestion
var FLOAT32_HALF_PI = float64ToFloat32( 1.5707963267948966 );
```

You want to use the double-precision value here, as it will be converted to single-precision.